src/main/archetype/core/pom.xml

```diff
@@ -76,6 +73,14 @@
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.compendium</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.service.component.annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.service.metatype.annotations</artifactId>
+        </dependency>
         <!-- Other Dependencies -->
```

Comment:
Explicitly adding the versioning annotations from https://mvnrepository.com/artifact/org.osgi/org.osgi.annotation/6.0.0 makes sense as well. The Bnd ones are deprecated now.